### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by chinmayic**

### CODE REVIEW

File: docs/index.html [hunk 1]
```diff
@@ -705,17 +712,16 @@ <h1 id='api-usage-and-access-guidelines'>API usage and access guidelines</h1>
...
-
+<pre class="highlight plaintext"><code>curl -v --cacert {path}/rootCA.crt 'https://{odimra_host}:{port}/redfish/v1'
+</code></pre>
 <p>{path} is where you have generated certificates during the deployment of the resource aggregator.</p>
 
 <p><strong>2.</strong> If you are running curl commands on a different server, perform the following steps to provide the rootCA.crt file.</p>
 
-<p>a. Navigate to <code>~/ODIM_v1.0/configuration/Odim/certificates</code> on the server where the resource
-         aggregator is deployed.</p>
+<p>a. Navigate to <code>~/ODIM_v1.0/configuration/Odim/certificates</code> on the server where the resource aggregator is deployed.</p>
```
Comment (ashraf-vazeer) [inline docs/index.html:725]: Can you check if there is version with ODIM folder?
that is,  ODIM_v1.0 or it should be just ODIM?

**Review by ashraf-vazeer — CHANGES_REQUESTED** [review]
Please address the comment

**Review by ashraf-vazeer — APPROVED** [review]
Look good to me

**Review by rahulroshan-kachchap — APPROVED** [review]
LGTM

**Pull request merged**